### PR TITLE
CI 

### PR DESCRIPTION
**Pull request by jaime-ez**
Hi @yasserf, github actions should be working ok.

I didn't push directly becasue on github I can not access the settings menu of this repo, and before releasing we need to make sure that the NPM_TOKEN is present on github secrets.

Let me know if it is there, and then we can merge and when publishing the new tag it should be published to npm :)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  release:
+    if: startsWith(github.ref, 'refs/tags/v')
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    # Setup .npmrc file to publish to npm
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
```
Comment (yasserf) [inline .github/workflows/build.yml:36]: Node 12

**Comment by yasserf** [comment]
Other than node 10 looks okay.

Interested to see how the server works, as it’s not the standard npm publish script.

**Comment by jaime-ez** [comment]
@yasserf the release jpb failed due to the missing NPM_TOKEN on github secrets (I guess). Can you add it please? I don't have access to this repo settings.

https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets

**Comment by jaime-ez** [comment]
@yasserf any chance you can take a look at the NPM_TOKEN on github secrets? Thanks

**Pull request merged**